### ISSUE TITLE
Add a proper Section on Contact US

### ISSUE THREAD

**Issue by deepeshgupta3257** [Opentek-Org/opentek#234]
**Is your feature request related to a problem? Please describe.**
Its not a problem but to have a particular section of the contact page makes easier for the user to contact instead of looking in the footer section of the website.
**Describe the solution you'd like**
Adding a proper section of the contact_us well make user directly to post their queries related to the content or any thing else.

So, please assign me this issue under GSSOC'23 so, that I can start working on this issue

**Comment by AnirudhPanda** [comment]
sorry for the delay in the response

do you still want to take this up @deepeshgupta3257

**Comment by deepeshgupta3257** [comment]
Yes I can take this issue. You can assign me this issue I will complete as soon as possible. So, please assign me this issue so that I can start working on it.

**Comment by AnirudhPanda** [comment]
> Yes I can take this issue. You can assign me this issue I will complete as soon as possible. So, please assign me this issue so that I can start working on it.

happy contributing!!!
please join https://discord.com/channels/1099745007172329592/1109164492287393954
